### PR TITLE
Basic Auth

### PR DESCRIPTION
**Pull request by christophgockel**

### CODE REVIEW

File: config/usage_help.go [hunk 1]
```diff
@@ -12,12 +12,14 @@ func UsageHelp() string {
...
+		"    -h, --host               specify the target host (and optional port)\n"+
+		"                             (default: %s)\n"+
+		"    -t, --threads            define number of parallel threads\n"+
+		"                             (default: %d)\n"+
+		"    -o, --out FILE           specify target FILE to write results to\n"+
+		"    -e, --endless            continuously repeat content of FILE\n"+
+		"                             (needs to be stopped with Ctrl+C)\n"+
+		"    -c USER:PASS             specify username and password for basic auth\n"+
+		"    --credentials USER:PASS\n"+
+		"        --help               show this usage text", DEFAULT_HOST, DEFAULT_NUMBER_OF_ROUTINES)
```
Comment (gemcfadyen) [inline config/usage_help.go]: Hmmm no test for this - should we have added one?

Comment (christophgockel) [inline config/usage_help.go]: There are no tests for that output. It's usually just tested by invoking `goony --help` and verifying that it works.

Comment (gemcfadyen) [inline config/usage_help.go]: 👍

**Comment by christophgockel** [comment]
Merged manually.

**Pull request closed without merging**